model/src/main/kotlin/SourceCodeOrigin.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model
+
+/**
+ * An enumeration of supported source code origins.
+ */
+enum class SourceCodeOrigin(val value: String) {
+    /**
+     * Source code from the version control system.
+     */
+    VCS("vcs"),
+
+    /**
+     * Source code from the distributed source code artifact.
+     */
+    ARTIFACT("artifact");
+
+    override fun toString() = value
```

Comment:
What's the `toString()` override / the value used for? Only for serialization? Can we omit both and live with the default `toString()` / `name()` implementation for `enums`?

Reply:
IMO it was nicer to use `vcs` and `artifact` instead of `VCS` and `ARTIFACT`.
But I'm ok with using upper case.

Reply:
> But I'd be ok with using upper case.

This doesn't seem to be done here. (And don't forget to also adjust the spelling in the commit message then.)